app.rb

```diff
@@ -1,8 +1,17 @@
 require 'sinatra/base'
+require './lib/peep'
+require './database_connection_setup'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
```

Comment:
You're missing a route here. I notice you have a view file called `index.erb` that isn't being used.

To have your user arrive at that page, you need something like this:
```ruby
  get '/' do
    erb :index
  end
```